### ISSUE TITLE
Automated code review tools missing

### ISSUE THREAD

**Issue by slikts** [joho/awesome-code-review#23]
There's quite a few automated code review or continuous inspection tools missing from this list:

  - [SonarQube](https://www.sonarqube.org/)
  - [Code Climate](https://codeclimate.com/quality/)
  - [Codebeat](https://codebeat.co/)
  - [Codacy](https://www.codacy.com/)
  - [DeepScan](https://deepscan.io/)
  - [DeepCode](https://www.deepcode.ai/)
  - [Scrutinizer](https://scrutinizer-ci.com/)

The only one currently listed is Sider, and the list is long enough that I'd argue it warrants a separate section.

I've also set up an [example repo ](https://github.com/slikts/code-examples)(with a focus on JS) to compare the results of the tools.

**Comment by joho** [comment]
Good point, I'm going to have a bit of a rethink on the tools section and what criteria to include or not include.

I do really like your example repo and think it could be a great resource to link as a standalone thing - the great automated code inspection tool shootout.

If you rounded out the readme in the repo to give more context I'd definitely add it to the list.

**Comment by cimfalab** [comment]
@slikts 
Thanks a ton for mentioning DeepScan! :heart:

**Comment by villelaitila** [comment]
Softagram is also missing. It is the only tool to automatically produce explanatory visualization about code changes.

**Comment by villelaitila** [comment]
> Softagram is also missing. It is the only tool to automatically produce explanatory visualization about code changes.

Adding links also here: 
Softagram: https://softagram.com
Softagram GitHub app: https://github.com/apps/softagram

**Comment by nimit156** [comment]
Hello,

I am sure you must have carried a hardcore research for the listing the most productive tools. I would suggest to add one more to the list : [Visual Expert](https://www.visual-expert.com/EN/lp-ve-2019-source_nov860ve.html?single) 
Visual Expert is already in use by small and large businesses in UK and USA. Also, the latest version with more code analyzing features have been released.
